desktop/core/src/desktop/settings.py

```diff
@@ -815,3 +815,14 @@ def tracer():
 except ValueError:
   from desktop.monkey_patches import monkey_patch_md5
   monkey_patch_md5(MODULES_TO_PATCH)
+
+################################################################
+# SLACK API Configurations
+################################################################
+# Warning: Do not save your secrets here, instead export them as environment variables and read here
+
+# use your keys 
+SLACK_CLIENT_ID=''
+SLACK_CLIENT_SECRET=''
+SLACK_VERIFICATION_TOKEN=os.environ.get('SLACK_VERIFICATION_TOKEN')
+SLACK_BOT_USER_TOKEN=os.environ.get('SLACK_BOT_USER_TOKEN')
```

Comment:
To offer as variable in the [[slack]] ini section

Reply:
Done